src/pages/Games.js

```diff
@@ -1,20 +1,16 @@
-import "../styles/pages/Games.css"
+// import "../styles/pages/Games.css"
```

Comment:
Any reason for commenting this?

Reply:
Yes, its empty...

Reply:
It's counterpart, activities.css is empty too, and it isn't included in activities.js. so i removed it here too.